### ISSUE TITLE
[Feedback] code blocks styling

### ISSUE THREAD

**Issue by LordGhostX** [acid-info/logos-docusaurus-plugins#72]
Do you think we can make the code blocks more noticeable for users unfamiliar with the docs?

<img width="754" alt="Screenshot 2023-06-19 at 12 55 32" src="https://github.com/acid-info/logos-docusaurus-plugins/assets/47832826/fc0573b4-a89e-4f5a-9adc-1985012cdb10">

https://waku-guide-git-add-nwaku-guides-logosdao.vercel.app/guides/nwaku/run-docker#get-docker-image

<img width="805" alt="Screenshot 2023-06-19 at 12 56 49" src="https://github.com/acid-info/logos-docusaurus-plugins/assets/47832826/83ed480b-f914-45e4-af1b-0f50efa8b426">

https://waku-guide-git-add-nwaku-guides-logosdao.vercel.app/guides/nwaku/configure-discovery#configure-dns-discovery

Here are some suggestions:

- Changing the font/code styling to a different theme
- Adding a background color that's different from the page, examples: [Vue](https://vuejs.org/guide/essentials/application.html), [GitHub](https://github.com/waku-org/nwaku/blob/master/docs/operators/how-to/build.md), [MUI](https://mui.com/material-ui/react-button/#basic-button)

**Comment by amirhouieh** [comment]
@LordGhostX good point! We should also make sure that code blocks font-family is monospace.

**Comment by jinhojang6** [comment]
Font: https://github.com/acid-info/logos-docusaurus-plugins/commit/10e09bd9de3fb82ae07be173d2ca4ce4c46cd5f7
monospace

**Comment by amirhouieh** [comment]
- [ ] apply monospace 
- [ ] for the default preset we apply Dracula for dark and  vsLight for the light mode
- [ ] then design review and check, if still not contrast enough then we go next 
- [ ] add background color to default code blocks (opacity 0.8)
